templates/artist/artist_detail.html

```diff
@@ -200,6 +226,69 @@ <h1 class="header-font">{{ artist.name }}</h1>
 {% block extrajs %}
     <script src="https://checkout.stripe.com/checkout.js"></script>
     <script type="text/javascript">
+
+    <script type="text/javascript" src="{% static "js/vendor/smooth-scroll.js" %}"></script>
+    <script type="text/javascript">
+
+
+
+        var doughnutData = [
+                {
+                    value: {{ campaign.fans_percentage }},
+                    color: "#46BFBD",
+                    highlight: "#5AD3D1",
+                    label: "Fan %"
+                },
+                {
+                    value: {{ campaign.artist_percentage }},
+                    color:"#F7464A",
+                    highlight: "#FF5A5E",
+                    label: "Artist %"
+                }
+
+            ];
+
+        var randomScalingFactor = function(){ return Math.round(Math.random()*100)};
+        var lineChartData = {
+            labels : ["1x","2x","3x","4x","5x"],
```

Comment:
Okay, this makes more sense to me now. However, I did kind of like the appearance of the curve before. What do you think of changing the labels and percentage_roi to match 0.5x, 1x, 2x, 3x, 5x, 10x?